### ISSUE TITLE
Tests fail (with DateTime 1.34?)

### ISSUE THREAD

**Issue by eserte** [LibreCat/Catmandu-Fix-Date#4]
My smokers started to report fails with Catmandu-Fix-Date-0.0129:

```
#   Failed test at t/Catmandu-Fix-end_year.t line 22.
#     Structures begin differing at:
#          $got->{end_year} = '2016-12-31T23:59:60Z'
#     $expected->{end_year} = '2017-01-01T00:00:00Z'
# Looks like you failed 1 test of 5.
t/Catmandu-Fix-end_year.t ......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/5 subtests 
```

Statistical analysis suggests that the fail is caused by a change in DateTime 1.34 (FYI @autarch ):

```
****************************************************************
Regression 'mod:DateTime'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    8971266385884729.00
[1='eq_1.26']        -0.0000          0.0000      -2.80
[2='eq_1.28']        -0.0000          0.0000      -1.95
[3='eq_1.34']        -1.0000          0.0000    -8718503788656084.00

R^2= 1.000, N= 50, K= 4
****************************************************************
```

**Comment by autarch** [comment]
Looks like the new leap second at the end of 2016 is causing the problem.

**Comment by nicolasfranck** [comment]
Should be fixed in commit https://github.com/LibreCat/Catmandu-Fix-Date/commit/94363d18b1b37d044a9bfc13c9d7737612abab0d.

See version https://github.com/LibreCat/Catmandu-Fix-Date/tree/0.0130
